wagtail/users/templatetags/wagtailusers_tags.py

```diff
@@ -98,9 +97,9 @@ def format_permissions(permission_bound_field):
                 custom_perms.append(
                     {
                         "perm": perm,
-                        "name": re.sub(
-                            f"{perm.content_type.name}$", "", perm.name, flags=re.I
-                        ).strip(),
+                        "name": "Can view"
+                        if permission_action == "view"
```

Comment:
I don't think this will solve the problem. Have another read of the issue in full. It explains that we need to try to resolve the model's verbose name here, with some discussion about different approaches. 

Hard-coding a specific label is probably not what we want to do.